test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Wide.Tests.ps1

```diff
@@ -1,35 +1,40 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 Describe "Format-Wide" -Tags "CI" {
+    BeforeAll {
+        1..10 | ForEach-Object { New-Item -Path ("TestDrive:\Testdir{0:00}" -f $_) -ItemType Directory }
+        1..10 | ForEach-Object { New-Item -Path ("TestDrive:\TestFile{0:00}.txt" -f $_) -ItemType File }
+    }
 
     It "Should have the same output between the alias and the unaliased function" {
-        $nonaliased = Get-ChildItem | Format-Wide
-        $aliased    = Get-ChildItem | fw
+        $nonaliased = Get-ChildItem $TestDrive | Format-Wide
+        $aliased = Get-ChildItem $TestDrive | fw
```

Comment:
Please remove the alias and use full name.

My understanding @SteveL-MSFT request is that we should move `Get-ChildItem $TestDrive` to `BeforeAll` and reuse it in all `It` blocks:
```c#
$pathList = Get-ChildItem $TestDrive
...
$aliased = $pathList | Format-Wide 
```